samples/bluetooth/periodic_adv_rsp/src/main.c

```diff
@@ -295,7 +295,14 @@ int main(void)
 	}
 
 	while (num_synced < MAX_SYNCS) {
-		err = bt_le_scan_start(BT_LE_SCAN_PASSIVE, device_found);
+		struct bt_le_scan_param scan_param = {
+			.type = BT_LE_SCAN_TYPE_PASSIVE,
+			.options = BT_LE_SCAN_OPT_FILTER_DUPLICATE,
+			.interval = per_adv_params.interval_min,
+			.window = per_adv_params.interval_min,
```

Comment:
Scan interval and window are units of 0.625 ms, whereas the periodic advertising interval are in units of 1.25ms. 

Is this intentional? If so, please add a comment to explain why we are setting these to the half the value of the PA interval.

Reply:
thanks for pointing it out. Those values should be double.

Reply:
reused the macros for contiuous scanner. https://github.com/zephyrproject-rtos/zephyr/pull/70044